docs/docusaurus/docs/tutorials/quickstart/quickstart.md

```diff
@@ -1,137 +1,96 @@
 ---
-title: Quickstart with GX
+title: Great Expectations Quickstart
 tag: [tutorial, getting started]
 ---
-# Quickstart with Great Expectations
-
 import Prerequisites from '/docs/components/_prerequisites.jsx'
 import SetupAndInstallGx from '/docs/components/setup/link_lists/_setup_and_install_gx.md'
 import DataContextInitializeInstantiateSave from '/docs/components/setup/link_lists/_data_context_initialize_instatiate_save.md'
 
-Few things are as daunting as taking your first steps with a new piece of software.  This guide will introduce you to GX and demonstrate the ease with which you can implement the basic GX workflow. We will walk you through the entire process of installing GX, connecting to some sample data, building your first Expectation based off of an initial Batch of that data, validating your data with that Expectation, and finally reviewing the results of your validation.
-
-Once you have completed this guide you will have a foundation in the basics of using GX.  In the future you will be able to adapt GX to suit your specific needs by customizing the execution of the individual steps you will learn here.
-
-## Prerequisites
-
-<Prerequisites requirePython = {true} requireInstallation = {false} requireDataContext = {false} requireSourceData = {null} requireDatasource = {false} requireExpectationSuite = {false}>
-
-- The ability to install Python packages with pip.
-- A working internet browser.
-
-</Prerequisites> 
-
-## Overview
-
-With GX you can get up and running with just a few lines of code.  The full process you'll be using will look like:
-
-```bash title="Terminal input"
-pip install great_expectations
-```
-
-```python name="tutorials/quickstart/quickstart.py all"
-```
-
-In the following steps we'll break down exactly what is happening here so that you can follow along and perform a Validation yourself.
-
-
-## Steps
-
-### 1. Install GX and set up your code environment
-
-#### 1.1 Install GX using pip
+# Great Expectations Quickstart
 
-Starting from an empty base directory inside a Python virtual environment, we use pip to install Great Expectations:
+Use this quickstart to install Great Expectations, connect to sample data, build your first Expectation, validate your data, and review the validation results. This is a great place to start if you're new to Great Expectations and aren't sure if it's the right solution for you or your organization. 
 
-```bash title="Terminal input"
-pip install great_expectations
-```
+:::note Great Expectations Cloud
 
-When you run this command from the terminal you will see `pip` go through the process of installing GX and it's related dependencies.  This may take a moment to complete.
+This quickstart introduces you to the open source Python and command line version of Great Expectations. A Cloud interface will soon be available to simplify collaboration between data teams and domain experts.
```

Comment:
Why are we talking about the command line at all? Isn't that a path we DONT want new users going done?

Reply:
yes agreed, we don't need to mention the command line.